akka-docs/src/main/paradox/typed/persistence-active-active.md

```diff
@@ -283,4 +285,12 @@ and then enable direct replication through `withDirectReplication()` on @apidoc[
 
 The "event publishing" feature publishes each event to the local system event bus as a side effect after it has been written, 
 the `ActiveActiveShardingDirectReplication` actor subscribes to these events and forwards them to the replicas allowing them
-to fast forward the stream of events for the origin replica. (With additional potential future support in journals for fast forwarding [#29311](https://github.com/akka/akka/issues/29311)). 
\ No newline at end of file
+to fast forward the stream of events for the origin replica. (With additional potential future support in journals for fast forwarding [#29311](https://github.com/akka/akka/issues/29311)). 
+
+## Journal Support
+
+For a journal plugin to support active active it needs to store and read metadata for each event if it is defined in the @apiref[PersistentRepr]
+ `metadata` field. To attach the metadata after writing it, `PersistentRepr.withMetadata` is used.
+ 
+For a snapshot plugin to support active active it needs to store and read metadata for the snapshot if it is defined in the @apiref[akka.persistence.SnapshotMetadata] `meta` field. 
+To attach the metadata when reading the snapshot the `akka.persistence.SnapshotMetadata.apply` factory overload taking a `meta` parameter is used.
```

Comment:
Is that correct that on one side we call it `metadata` and on the other side it's `meta`? Shouldn't we take the opportunity to align the naming?

Reply:
I agree, I'll rename it right away in this pr